sdk/cmake/CMakeMacros.cmake

```diff
@@ -642,18 +626,33 @@ function(set_module_type MODULE TYPE)
         else()
             message(STATUS "${MODULE} has no base address")
         endif()
-    elseif((${TYPE} STREQUAL kernelmodedriver) OR (${TYPE} STREQUAL wdmdriver) OR (${TYPE} STREQUAL kerneldll))
-        set_image_base(${MODULE} 0x00010000)
+    elseif(TYPE IN_LIST KERNEL_MODULE_TYPES)
+        # special case for kernel
+        if (TYPE STREQUAL kernel)
+            if (MSVC)
+                set_image_base(${MODULE} 0x00400000)
+            else()
+                set_image_base(${MODULE} 0x80800000)
```

Comment:
The `if(GDB AND NOT CLANG)` case used to set `0x00800000`. Is that intended?

Reply:
Yes. Actually, 0x00400000 works for everyone now. This is a remnant of old RosBE.